scripting/get5/matchconfig.sp

```diff
@@ -392,7 +392,7 @@ static bool LoadMatchFromKv(KeyValues kv) {
   }
 
   if (AddSubsectionKeysToList(kv, "maplist", g_MapPoolList, PLATFORM_MAX_PATH) <= 0) {
-    LogMessage("Failed to find \"maplist\" section in config, using fallback maplist.");
+    MatchConfigFail("Failed to find \"maplist\" section in config, using fallback maplist.");
     LoadDefaultMapList(g_MapPoolList);
```

Comment:
I'm not sure I know what the precise error is yet (I haven't looked through the existing code yet), but I don't think this is the right change. 

The `maplist` list is meant to be optional, `MatchConfigFail` is only meant to be called when it explicitly fails.

Reply:
Ah I see, would `PrintToServer` be of better use?

Reply:
I'm not sure what the problem with LogMessage is here. 

> * Fix output (translation doesn't exist, use standard output)

Is there something in error logs that make you think that? I don't see any issue with the current usage.

Reply:
Yeah I think you're right, I'll revert this change